src/encoded/batch_download.py

```diff
@@ -84,6 +84,7 @@ def includeme(config):
     ('Platform', ['files.platform.title']),
```

Comment:
@caseylitton  What code change is still required?

Reply:
I suggest slacking people when you need to remind them about code review

Reply:
@caseylitton Acknowledged